content/docs/1.2.0/snapshots-and-backups/backup-and-restore/_index.md

```diff
@@ -1,4 +1,10 @@
 ---
 title: Backup and Restore
 weight: 2
----
\ No newline at end of file
+---
+
```

Comment:
I think the statement here can be as a note by using `>` markdown syntax because it is describing the change between before and after 1.2.0.

Below this new note, you can use a TOC of the subpages to enrich this page like below monitoring page.

![image](https://user-images.githubusercontent.com/899492/129155699-c7c252df-7c8a-42da-bffd-693b803f2f92.png)